test/base001/expected

```diff
@@ -0,0 +1,27 @@
+Type checking ./base001.idr
+exit1: Executing cmd 'exit 1'
+exit1: raw exitStatus = 256
+exit1: WEXITSTATUS(exitStatus) = 1
+exit1: Executing cmd './does-not-exist'
+exit1: raw exitStatus = 32512
+exit1: WEXITSTATUS(exitStatus) = 127
+-------------------------------------------
+Executing 'exit 1'
+raw status = 256
+exit status = 1
+True
+
+-------------------------------------------
+Executing './exit1'
+raw status = 256
+exit status = 1
+True
+
+-------------------------------------------
+Executing './does-not-exist'
+raw status = 32512
+exit status = 127
+True
+
+Removed: 00base001-idx.ibc
+Removed: /Users/steshaw/Projects/steshaw/idris/test/base001/base001
```

Comment:
In case you didn't notice it, the expected output contains information specific to your system. That is, it contains an absolute path.